app/controllers/people_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+    @people_hash = {}
+    @people.each do |person|
+      @tasks_completed = person.tasks.where(completed: true)
+      @people_hash[person.name] = @tasks_completed.count
+    end
+    @people_hash
+  end
+
+  def show
+    @person = Person.find(params[:id])
+    @tasks = @person.tasks.where(person_id: @person.id)
```

Comment:
Because the tasks are associated with the person via the model, you do not need to add this where method call. It is already only going to include the tasks associated with this specific person.